src/category_theory/category/Pointed.lean

```diff
@@ -79,6 +80,16 @@ between them. -/
   hom_inv_id' := Pointed.hom.ext _ _ e.symm_comp_self,
   inv_hom_id' := Pointed.hom.ext _ _ e.self_comp_symm }
 
+instance monoidal_category : monoidal_category Pointed.{u} :=
+{ tensor_obj := λ X Y, ⟨X × Y, (X.point, Y.point)⟩,
+  tensor_hom := λ W X Y Z f g, ⟨prod.map f.to_fun g.to_fun, prod.ext f.map_point g.map_point⟩,
+  tensor_unit := ⟨punit, punit.star⟩,
+  associator := λ X Y Z, iso.mk (equiv.prod_assoc _ _ _) rfl,
+  left_unitor := λ X, iso.mk (equiv.punit_prod _) rfl,
+  right_unitor := λ X, iso.mk (equiv.prod_punit _) rfl }
+
+@[simp] lemma point_tensor (X Y : Pointed) : (X ⊗ Y).point = (X.point, Y.point) := rfl
```

Comment:
Might be nice to have the analogous this for the unit, and maybe simp lemmas for the tensor product of morphisms?

Reply:
Also for the unitors actually - remember that all these are data so it's nice to have lemmas describing them so you don't need to unfold the instance

Reply:
Can you point to an existing `monoidal_category` instance with such refl lemmas?

Reply:
From a quick github search: https://github.com/leanprover-community/mathlib/blob/418373e885f1adfc02882efed6d04ba63ebfe20b/src/algebra/category/Module/monoidal.lean#L185

Reply:
If it's helpful for you to have more examples, there's another in https://github.com/leanprover-community/mathlib/blob/3225926b9f2c1e31b4d4329aec72f69555270282/src/category_theory/monoidal/types.lean#L31

I can find more if you'd like!

Reply:
`Type` was very helpful, thanks! I could literally copy everything over and it worked.